instrumentation/logback/logback-access-jetty-12.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/logback/access/jetty/v12_0/LogbackAccessInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.logback.access.jetty.v12_0;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class LogbackAccessInstrumentationModule extends InstrumentationModule {
+  public LogbackAccessInstrumentationModule() {
+    super("logback-access-jetty-12", "logback-access-jetty-12-1.0");
```

Comment:
```suggestion
    super("logback-access-jetty-12", "logback-access-jetty-12.0");
```

Reply:
i tried but it will fail at `check-javaagent-suppression-keys.sh`. hence i updated it to 
```
    super("logback-access-jetty", "logback-access-jetty-12.0");
```